evm/src/executor/builder.rs

```diff
@@ -152,6 +152,13 @@ impl ExecutorBuilder {
         self
     }
 
+    /// Enables coverage collection
+    #[must_use]
+    pub fn with_coverage(mut self) -> Self {
+        self.inspector_config.coverage = true;
+        self
+    }
```

Comment:
considering how this will be configured, perhaps we should make this `set_coverage(mut self, bool)` so we can easily chain this to the `Builder`